typescript/cli/src/tests/warp/warp-rebalancer.e2e-test.ts

```
@@ -0,0 +1,1614 @@
+        console.log(chunk);
+
+        if (chunk.includes(expectedLogs[0])) {
+          expectedLogs.shift();
+
+          if (!expectedLogs.length) {
+            resolve(void 0);
+            break;
+          }
+        }
+      }
+    }).finally(() => {
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)